hazelcast/src/main/java/com/hazelcast/core/LifecycleEvent.java

```diff
@@ -42,6 +42,7 @@
         SHUTDOWN,
         MERGING,
         MERGED,
+        MERGE_FAILED,
```

Comment:
`LifecycleEvent` is not serialized. It should be completely local event. Relying ordinals unless explicitly defined should be considered as bug, I think.